forum/users/views.py

```diff
@@ -1,11 +1,108 @@
+from os import environ
+
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.shortcuts import get_object_or_404
 from rest_framework_simplejwt.views import (
     TokenObtainPairView as BaseTokenObtainPairView,
-    TokenRefreshView as BaseTokenRefreshView
+    TokenRefreshView as BaseTokenRefreshView,
+)
+from rest_framework.generics import GenericAPIView
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.exceptions import NotFound
+
+from users.models import CustomUser
+from forum.tasks import send_email_task
+from forum.utils import build_email_message
+from .models import PasswordResetModel
+from .serializers import (
+    PasswordResetRequestSerializer,
+    PasswordResetSerializer
 )
+from .throttling import PasswordResetThrottle
+
 
 class TokenObtainPairView(BaseTokenObtainPairView):
     throttle_scope = 'token_obtain'
 
 
 class TokenRefreshView(BaseTokenRefreshView):
     throttle_scope = 'token_refresh'
+
+
+class PasswordResetRequestView(GenericAPIView):
+    serializer_class = PasswordResetRequestSerializer
+    permission_classes = []
```

Comment:
Do we need to specify "AllowAny", considering that we have default permission class "IsAuthenticated"?

Reply:
I replaced `permission_classes` with an empty list, so unauthorized users are allowed to interact with this resource.